aptos-move/framework/aptos-stdlib/sources/table_with_length.move

```diff
@@ -73,6 +73,17 @@ module aptos_std::table_with_length {
         }
     }
 
+    /// Insert the pair (`key`, `value`) if there is no entry for `key`.
+    /// update the value of the entry for `key` to `value` otherwise
+    public fun upsert<K: copy + drop, V: drop>(table: &mut TableWithLength<K, V>, key: K, value: V) {
```

Comment:
Would prefer a real name, like 'insert_or_update`. Also all our map-like types should have the same API (Table, SimpleMap).

Can we please be a bit more careful adding things to the standard library?